quickstart-templates/workload-identity-creates-workload-identities/README.md

```diff
@@ -0,0 +1,70 @@
+# Create Workload Identity Federation that can create other workload identity federations
+
+## The Idea
+
+Restrict and limit the need to use high human privileges to create resources.
+
+## Example scenario
+
+A high-privilege user creates an initial, bootstrap workload identity federation token between Azure and a certain GitHub repository. The workload in this repository is then allowed to access Microsoft Entra to create further workload identity federation tokens and assign them to other repositories.
```

Comment:
Does this mean a user that is a member of a privileged role or a user that has consented to a high privilege permission?

Reply:
Either works? Actually, now that I think of this and your previous question, it's probably worth being more accurate and use the correct terms etc. Can you help with that? How could I phrase this part?

F.ex. while I deployed to root structures I ran `az role assignment list --scope "/" --query "[?roleDefinitionName=='User Access Administrator'].{Username:principalName, ObjectId:objectId}" --output table` returned my user name. Consequently I started to think that OK, I probably need to set up some small things to set up more from a GitHub and then delegate further to smaller privileged GitHub actions further things.

Reply:
I think there are multiple things going on here, that probably need more explanation in the usage sections and maybe in the template comments.  At this point you probably just need something high level.

The multiple things I'm thinking are:
1. **Delegated perms to create the "creator" workload identity (part 1):** Just to "start" the picture, although this is somewhat transparent for most users. For delegated permissions, the calling application must also have the necessary privileges. In this case, Azure PS or Azure CLI has been pre-authorized with those permissions.  They're also pre-authorized for full delegated access to ARM - so that both these apps are delegated to access ARM APIs as the signed-in user (they have whatever privileges the signed in user has).
2. **Delegated perms to create the "creator" workload identity (part 2):** This is the high-privileged user that you mention. They need to have the privileges:
   - to create apps/service principals (i.e. create the creator workload identity) AND to grant that workload identity the permissions it needs to create other "repo" workload identities.  These privileges are normally assigned to users through Entra RBAC roles.
   - if the repo workload identities (that are created via the creator workload identity) need access to Azure resources, then the creator workload identity needs to be assigned the privileges that allows it to create managed identities (for FIC) and assign (scoped) Azure roles to repo workload identities. My assumption here is that this signed-in user's (privileged Azure RBAC permissions/roles) are scoped to an Azure resource group, which allow them to assign the necessary Azure permissions/roles to the creator workload identity.
3. **Permissions in GitHub and GitHub actions**: I think you allude to this in your last sentence. This is about how the workload identities' access is scoped in GitHub.  I have less familiarity with this and not sure if there is any repo hierarchy, but I could imagine that the creator workload identity is scoped to an organization, which would allow it to create workload identities scoped to all repos in the organization. However, like I say, you probably have a better idea on this than me.